proto/cosmos/authz/v1beta1/tx.proto

```diff
@@ -6,57 +6,59 @@ import "gogoproto/gogo.proto";
+  // Grant grants the provided authorization to the grantee on the granter's
   // account with the provided expiration time.
-  rpc GrantAuthorization(MsgGrantAuthorizationRequest) returns (MsgGrantAuthorizationResponse);
+  rpc Grant(MsgGrantRequest) returns (MsgGrantResponse);
 
-  // ExecAuthorized attempts to execute the provided messages using
+  // Exec attempts to execute the provided messages using
   // authorizations granted to the grantee. Each message should have only
   // one signer corresponding to the granter of the authorization.
-  rpc ExecAuthorized(MsgExecAuthorizedRequest) returns (MsgExecAuthorizedResponse);
+  rpc Exec(MsgExecRequest) returns (MsgExecResponse);
 
-  // RevokeAuthorization revokes any authorization corresponding to the provided method name on the
+  // Revoke revokes any authorization corresponding to the provided method name on the
   // granter's account that has been granted to the grantee.
-  rpc RevokeAuthorization(MsgRevokeAuthorizationRequest) returns (MsgRevokeAuthorizationResponse);
+  rpc Revoke(MsgRevokeRequest) returns (MsgRevokeResponse);
 }
 
-// MsgGrantAuthorizationRequest grants the provided authorization to the grantee on the granter's
+// MsgGrantRequest grants the provided authorization to the grantee on the granter's
 // account with the provided expiration time.
-message MsgGrantAuthorizationRequest {
+message MsgGrantRequest {
   string granter = 1;
   string grantee = 2;
 
   google.protobuf.Any       authorization = 3 [(cosmos_proto.accepts_interface) = "Authorization"];
   google.protobuf.Timestamp expiration    = 4 [(gogoproto.nullable) = false, (gogoproto.stdtime) = true];
 }
 
-// MsgExecAuthorizedResponse defines the Msg/MsgExecAuthorizedResponse response type.
-message MsgExecAuthorizedResponse {
+// MsgExecResponse defines the Msg/MsgExecResponse response type.
+message MsgExecResponse {
   cosmos.base.abci.v1beta1.Result result = 1;
 }
 
-// MsgExecAuthorizedRequest attempts to execute the provided messages using
+// MsgExecRequest attempts to execute the provided messages using
 // authorizations granted to the grantee. Each message should have only
 // one signer corresponding to the granter of the authorization.
-message MsgExecAuthorizedRequest {
+message MsgExecRequest {
   string   grantee                  = 1;
   repeated google.protobuf.Any msgs = 2;
 }
 
-// MsgGrantAuthorizationResponse defines the Msg/MsgGrantAuthorization response type.
-message MsgGrantAuthorizationResponse {}
+// MsgGrantResponse defines the Msg/MsgGrant response type.
+message MsgGrantResponse {}
 
-// MsgRevokeAuthorizationRequest revokes any authorization with the provided sdk.Msg type on the
+// MsgRevokeRequest revokes any authorization with the provided sdk.Msg type on the
 // granter's account with that has been granted to the grantee.
-message MsgRevokeAuthorizationRequest {
+message MsgRevokeRequest {
```

Comment:
```suggestion
message MsgRevoke {
```

Reply:
I thought we want to have `Request` suffix for all request argument types.